src/sentry/issues/ingest.py

```diff
@@ -1,10 +1,40 @@
 from __future__ import annotations
 
+import logging
+from datetime import datetime
+from hashlib import md5
+from typing import Any, Mapping, Optional, TypedDict, cast
+
+import sentry_sdk
+from django.conf import settings
+from django.db import transaction
+
+from sentry.constants import LOG_LEVELS_MAP
+from sentry.event_manager import (
+    GroupInfo,
+    _process_existing_aggregate,
+    _save_grouphash_and_group,
+    get_event_type,
+)
 from sentry.eventstore.models import Event
 from sentry.issues.issue_occurrence import IssueOccurrence, IssueOccurrenceData
+from sentry.models import GroupHash, Release
+from sentry.ratelimits.sliding_windows import Quota, RedisSlidingWindowRateLimiter, RequestedQuota
+from sentry.types.issues import GROUP_TYPE_TO_CATEGORY
+from sentry.utils import metrics
+
+issue_rate_limiter = RedisSlidingWindowRateLimiter(
+    **settings.SENTRY_ISSUE_PLATFORM_RATE_LIMITER_OPTIONS
+)
+# This should probably be configurable per team
+ISSUE_QUOTA = Quota(3600, 60, 5)
+GROUPHASH_IGNORE_LIMIT = 3
```

Comment:
do we want to start with this on? feel like let's keep it off until GA, could keep it on for EA if that makes sense.

Reply:
i see it's not implemented below so that's fine